deployment/Makefile

```diff
@@ -329,6 +329,35 @@ flake8:
 	@echo "Check flake 8"
 	@flake8 ../django_project
 
+update-translation-strings:
+	@echo
+	@echo "------------------------------------------------------------------"
+	@echo "Update string for translation."
+	@echo "------------------------------------------------------------------"
+	@docker-compose -p $(PROJECT_ID) exec uwsgi python manage.py makemessages
+
+compile-translation-strings:
+	@echo
+	@echo "------------------------------------------------------------------"
+	@echo "Compile string for translation."
+	@echo "------------------------------------------------------------------"
+	@docker-compose -p $(PROJECT_ID) exec uwsgi python manage.py compilemessages
+
+push-translation-source:
+	@echo
+	@echo "------------------------------------------------------------------"
+	@echo "Push translation source and translation to transifex."
```

Comment:
This help looks weird. I would avoid the second word 'translation'. Maybe just "push translations to transifex"

Reply:
ah thanks, perhaps I was too sleepy